style.css

```diff
@@ -0,0 +1,89 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Roboto&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  scroll-behavior: smooth;
+}
+
+body {
+  overflow-x: hidden;
+  max-width: 375px;
+  margin: auto;
+  height: auto;
+}
+
+nav {
+  margin-top: 30px;
+  display: flex;
+  justify-content: flex-end;
+}
+
+img {
+  width: 26px;
+  height: auto;
+}
+
+h1 {
+  font-family: "Inter", sans-serif;
+  font-style: bold;
+  font-weight: 700;
+  font-size: 45px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+}
+
+p {
+  font-family: "Inter", sans-serif;
+  margin-top: 12px;
+  font-style: normal;
+  font-weight: 390;
+  font-size: 20px;
+  line-height: 28px;
+  width: 400px;
+  text-align: center;
+}
```

Comment:
Your design 
![image](https://github.com/Mike47ip/mobilePortfolioWeb/assets/78251927/cb5f368b-e47f-4018-a382-b0d8a610ed7f)

Figma
![image](https://github.com/Mike47ip/mobilePortfolioWeb/assets/78251927/4be6b189-73ce-4da9-8932-7b8d45ae6614)

I would suggest that we add more spacing `right and left` similar to the Figma design. 👍